src/Integration.UnitTests/Binding/SolutionBindingOperationTests.cs

```diff
@@ -333,6 +335,60 @@ public void SolutionBindingOperation_Prepare_Cancellation_ProjectBindersAreNotCa
             CheckSaveWasNotCalled(vbConfigFile);
         }
 
+        [TestMethod]
+        public void SolutionBindingOperation_Prepare_Cancellation_DuringBindersPrepare_SomeProjectBindersAreNotCalled()
```

Comment:
Test from this comment:
https://github.com/SonarSource/sonarlint-visualstudio/pull/1391#discussion_r421412555